code/game/objects/items/storage/belt.dm

```diff
@@ -965,7 +966,17 @@
 		sure that we don't have to do any extra calculations.
 		*/
 		playsound(src, drawSound, 7, TRUE)
-		var/image/gun_underlay = image(icon, current_gun.base_gun_icon)
+		var/image/gun_underlay
+		if(gun_has_gamemode_skin)
+			switch(SSmapping.configs[GROUND_MAP].camouflage_type)
+				if("snow")
```

Comment:
are there defines for these?

Reply:
afaik all camo swapping instances are handled similar to this, but i could possibly missing something. not sure